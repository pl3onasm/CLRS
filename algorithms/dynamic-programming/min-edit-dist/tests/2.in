democracy
oligarchy

1 3 2 1 2 10
