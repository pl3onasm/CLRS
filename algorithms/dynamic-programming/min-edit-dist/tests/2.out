Minimum edit distance: 9

Optimal operation sequence:

Source:
  democracy
Replace d with o:
  o | emocracy
Replace e with l:
  ol | mocracy
Replace m with i:
  oli | ocracy
Replace o with g:
  olig | cracy
Replace c with a:
  oliga | racy
Copy r:
  oligar | acy
Replace a with c:
  oligarc | cy
Replace c with h:
  oligarch | y
Copy y:
  oligarchy | 
Target:
  oligarchy
